ddtrace/opentelemetry/span.go

```diff
@@ -45,6 +49,40 @@ func (s *span) SetName(name string) {
 	s.attributes[ext.SpanName] = strings.ToLower(name)
 }
 
+func stringifySpanEvents(evts []otelsdk.Event) (s string) {
+	for i, e := range evts {
+		if i == 0 {
+			s += marshalSpanEvent(e)
+		} else {
+			s += "," + marshalSpanEvent(e)
+		}
+	}
+	return s
+}
+
+func marshalSpanEvent(evt otelsdk.Event) string {
+	type ddSpanEvent struct {
+		Name           string                 `json:"name"`
+		Time_unix_nano int64                  `json:"time_unix_nano"`
+		Attributes     map[string]interface{} `json:"attributes,omitempty"`
+	}
```

Comment:
Do we need the type inside the function?

Reply:
i think this is on an older commit; I changed it so the type is at the package level. And anyway, it'll need to be at the package level since I'll use it for the span's `events` slice. :)